docs/contributing.md

```diff
@@ -1,58 +1,61 @@
-# Developer Contributor Guide
+# Contributor Guide
 
-We collaborate on code through git, hosted on GitHub. If you are a software engineer (web/mobile/backend), you should request write access to the codebase.
+Couchers.org is currently split into teams, among them: product (divided into backend, web, mobile, and devops), design, community, marketing, and support and moderation.
+
+This guide was written primarily for product team members to contribute to the codebase, but other contributors will find value in [Finding an issue](#finding-an-issue), [Detailing a task](#detailing-a-task), and [Project management](#project-management).
+
+First of all though, please take a look at our [public roadmap](https://github.com/orgs/Couchers-org/projects/4/views/7) to follow our big picture progress and promises. For more detail on feature development, check out the [contributor roadmap](https://github.com/orgs/Couchers-org/projects/4/views/4) to see where the team is heading.
 
 ## Finding an issue
 
-If you are looking for something to help with, have a look at the open issues! They are sorted by tags to help with filtering. Here are some examples:
+If you are a new contributor, we recommend starting with a [good first issue](https://github.com/orgs/Couchers-org/projects/4/views/6?sliceBy%5Bvalue%5D=good+first+issue) and then a [good second issue](https://github.com/orgs/Couchers-org/projects/4/views/6?sliceBy%5Bvalue%5D=good+second+issue). Otherwise, please look at [current open tasks](https://github.com/orgs/Couchers-org/projects/4/views/3?sliceBy%5Bvalue%5D=_noValue) or even [next sprint's open tasks](https://github.com/orgs/Couchers-org/projects/4/views/9?sliceBy%5Bvalue%5D=_noValue) if nothing is currently open. When you find a task you want to work on, leave a comment saying the same and assign the task to yourself.
+
+## Detailing a task
```

Comment:
I think this section is great, but I feel it might seem like a lot of process for someone who is new to contributing to the project. Maybe we can move it further down and/or do something else to make it clear that one doesn't have to go through the this whole part to push through their first issue? What do you think?